integrations/ollama/src/haystack_integrations/components/generators/ollama/chat/chat_generator.py

```diff
@@ -40,6 +41,7 @@ def __init__(
         timeout: int = 120,
         keep_alive: Optional[Union[float, str]] = None,
         streaming_callback: Optional[Callable[[StreamingChunk], None]] = None,
+        response_format: Optional[Union[Literal["", "json"], JsonSchemaValue]] = None,
```

Comment:
I understand that the type hint comes from the Ollama python client, but it is confusing to me.

I would change it to something more readable, like `response_format: Union[None, Literal['json'], JsonSchemaValue] = None`
- default is None (TODO: check if it has the same meaning as an empty string)
- you can pass "json" if you only want to ensure that the output format is a valid JSON
- `JsonSchemaValue` corresponds to Json Schemas, passed as dictionaries

I would add a similar but more refined explanation to the `response_format` docstring above.